documentation/work-instructions/user-management.md

```diff
@@ -0,0 +1,71 @@
+# Work instruction: User Management
+
+Using the Gateway User Interface, you can manage (create, read, update and delete) users. In this work instruction, you will learn how.
+
+
+## Prerequisites
+
+1. You're logged into the Gateway User Interface with an existing user
+
+
+## Step 1: Getting there
+
+1. Navigate to "Settings", at the bottom of the navigation
+2. Navigate to "Users", one of the tabs
+
+
+## Step 2: Creating a new user
+
+1. Click on "+ Add User"
+
+1. Fill in the *required* fields
+   1. Name*: the name of the new user
+   2. Email*: the email address of the new user
+   3. Organization*: the organization to which the new user should belong
+   4. Applications*: one or more applications to which the new user has access to
+   5. Password*: set an initial password for the user (retype it in the second field for verification)
+
+3. Optionally fill in the _optional_ fields
+   1. Description: optionally set a description of the user
+   2. Locale: optionally set the locale of the user (e.g. en-US)
+   3. Person: optionally add extra information about the person
+   4. Security Groups: optionally add Security Groups to this user to give them more rights
+
+4. Click on "Save" to complete the process
+
+> Tip: Organizations, Applications and Security Groups can also be managed within the Gateway User Interface.
+
+
+## Step 3: Editing an existing user
+
+1. Click on "-> Details" to navigate to the detail page of the to edit user
+
+2. Edit one or multiple fields in the form
+
+3. Click on "Save" to complete the process
+
+> Tip: click on "+ Add to dashboard" or "- Remove from dashboard" to toggle users as Dashboard Cards on the dashboard page!
+
```

Comment:
Maybe we should add an extra tip for the password fields:
`Password wil not be reset when saving the user if the fields are empty.`